src/pages/namespace/Events/Listeners/NoResult.tsx

```diff
@@ -0,0 +1,13 @@
+import { Calendar } from "lucide-react";
+
+const NoResult = ({ message }: { message: string }) => (
+  <div
+    className="flex flex-col items-center justify-center gap-1 p-10"
+    data-testid="listener-no-result"
+  >
+    <Calendar />
```

Comment:
I think you forgot to update the Calendar icon. When you search for `<Calendar />` in the `./src/pages/namespace/Events`folder, I find 4 currencies that might need to be updated.